tests/react-integration-test/check-japgolly/r/react-dropzone/src/main/scala/typingsJapgolly/reactDropzone/components/ReactDropzone.scala

```diff
@@ -30,11 +29,13 @@ object ReactDropzone {
     @scala.inline
     def accept(value: String | js.Array[String]): this.type = set("accept", value.asInstanceOf[js.Any])
     @scala.inline
-    def children(value: /* state */ DropzoneState => CallbackTo[Element]): this.type = set("children", js.Any.fromFunction1((t0: /* state */ DropzoneState) => value(t0).runNow()))
+    def children(value: /* state */ DropzoneState => Element): this.type = set("children", js.Any.fromFunction1(value))
     @scala.inline
     def disabled(value: Boolean): this.type = set("disabled", value.asInstanceOf[js.Any])
     @scala.inline
-    def getFilesFromEvent(value: /* event */ DropEvent => CallbackTo[js.Any]): this.type = set("getFilesFromEvent", js.Any.fromFunction1((t0: /* event */ DropEvent) => value(t0).runNow()))
+    def getFilesFromEvent(
+      value: /* event */ DropEvent => /* import warning: transforms.QualifyReferences#resolveTypeRef many Couldn't qualify Promise<Array<File | DataTransferItem>> */ _
```

Comment:
Is this warning something to be concerned about?

Reply:
Yeah, it's fine actually. When we added all the react integration tests we didn't do all the busy work with ensuring that all referenced types were actually available.